Toad-Labs/vc-frontend-static
# Translating emails / Job bank message

## Translating emails / Job bank message

### Description
Email is now translated from service. This is done through use of locale. Various dates are using locale to translate as well

### Additional Notes
[VCON-252 Translate Virtual Assistant to Assistant virtuel](https://decdvirtualconcierge.atlassian.net/browse/VCON-252?atlOrigin=eyJpIjoiMDhiY2I1ZmVjZjM3NGE1N2E4ZWUzZjY5ZDJmM2ZmYmIiLCJwIjoiaiJ9) is included in this change

### Test
1. View page in English and make sure all text corresponds
2. View page in French and make sure all text corresponds
3. Search through Inbox
4. *Chatbot messages aren't required to be translated as of yet as it exists outside of the VC front end
